### ISSUE TITLE
[Bug Report] Water physics

### ISSUE THREAD

**Issue by Nisutec** [ammaraskar/pyCraft#180]
After joining the server with the lib the player is not affected from water physics. Is this a bug or is it possible to activate this behavior.

![Problem](https://p52.p0.n0.cdn.getcloudapp.com/items/yAuYGGE6/2020-05-31_18.25.57.png?v=ca3b089179ab8110204e3230a8309578)

**Comment by MiniDigger** [comment]
you would need to implement such movement yourself

**Comment by Nisutec** [comment]
yes but how i can do this i can't find any docomentation about this topic on the docomentation.
How i can enable the normal physics of a player ?

**Comment by Skelmis** [comment]
You would need to implement additional packets to handle movement and world interaction I believe
